scala/scala.bzl

```diff
@@ -70,6 +64,7 @@ _resolve_deps = {
         allow_files = False),
 }
 
+# TODO not version specific
```

Comment:
Can you explain?

Reply:
Leftover, was relevant when trying to use multiple versions, I'll remove